.github/workflows/main.yml

```
@@ -74,4 +75,31 @@ jobs:
         uses: actions/upload-artifact@v1
         with:
           name: statements
-          path: /tmp/html
\ No newline at end of file
+          path: /tmp/html
+
+  mingw-windows:
+    runs-on: windows-latest
+    strategy:
+      matrix:
+        id: [1, 2, 3, 4, 5]
+    steps:
+      - uses: actions/checkout@v1
+      - name: Set up Python 3.5.4
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.5.4
+      - name: Install dependencies
+        run: |
+          python -m pip install --upgrade pip
+          pip install --user -r requirements.txt
+      - name: Print problem list
+        run: |
+          python ci_generate.py --show-list 5 ${{ matrix.id }}
+      - name: Run generate.py on gcc
+        run: |
+          python ci_generate.py 5 ${{ matrix.id }} --html --htmldir=D:/tmp/html
+      - name: Archive problem statements
```

```suggestion
      - name: Archive problem statements
```

よく考えたらここからいらなかった…(actionsが作ったhtmlをダウンロードできるようにするステップなんですが、ubuntuの方でアップロードしているので)

---

直しました